pipelines/build/common/build_base_file.groovy

```diff
@@ -968,6 +968,16 @@ class Builder implements Serializable {
                                             throw new Exception("[ERROR] Archive artifact timeout (${pipelineTimeouts.ARCHIVE_ARTIFACTS_TIMEOUT} HOURS) for ${downstreamJobName}has been reached. Exiting...")
                                         }
 
+                                        // Archive tap files as a single tar file
+                                        context.sh "find . -type f -name '*.tap' -exec tar -czf AQAvitTapFiles.tar.gz {} + "
```

Comment:
Given a sufficient number of `*.tap` files, `find` will run `tar` more than once, the result being that the archive only contains the last batch of `*.tap` files.

Reply:
Thanks @keithc-ca . You are correct. Will update